crates/bevy_reflect/bevy_reflect_derive/src/field_attributes.rs

```diff
@@ -0,0 +1,74 @@
+use quote::ToTokens;
+use syn::spanned::Spanned;
+use syn::{Attribute, Meta, NestedMeta};
+
+pub(crate) static REFLECT_ATTRIBUTE_NAME: &str = "reflect";
+pub(crate) static REFLECT_VALUE_ATTRIBUTE_NAME: &str = "reflect_value";
+pub(crate) static IGNORE: &str = "ignore";
+
+/// A container for reflection field configuration.
+#[derive(Default)]
+pub struct ReflectFieldAttr {
+    /// Determines if this field should be ignored.
+    pub ignore: Option<bool>,
```

Comment:
That's a good question. This could definitely be made into a standard `bool`. I'm guessing the original reasoning behind was either: (a) to be used directly for parsing or (b) to indicate the attribute's existence (or lack thereof). I'll see about switching it to a regular `bool`.